doc/Command_Index_User.tex

```diff
@@ -154,66 +147,126 @@ \subsubsection{Including scores}
-  firstsyllableinitial & the first letter of the first syllable of a score which is not the score initial & none\\
-  firstsyllable & the balance of the first syllable of the score & none
-\end{tabular}
+\macroname{\textbackslash grecreatedim}{\{\#1\}\{\#2\}\{\#3\}}{gregoriotex-spaces.tex}
+Macro to create one of Gregorio\TeX’s distances.  Used to initialize distances in a space configuration file.
 
-\macroname{\textbackslash grechangestyle}{\{\#1\}\{\#2\}[\#3]}{gregoriotex.sty and gregoriotex.tex}
-Command to change styling of a score element.
+\begin{argtable}
+  \#1 & string & The name of the distance to be changed.  See \nameref{distances} below.\\
+  \#2 & string & The distance in string format.  \textbf{Note:} You cannot use a length register for this argument.  You \emph{must} use a string because of the way that Gregorio\TeX\ handles spaces.\\
+  \#3 & 0 & Distance will not scale when staff size is changed.\\
+  & 1 & Distance will scale when staff size is changed.
+\end{argtable}
+
+\macroname{\textbackslash grechangedim}{\{\#1\}\{\#2\}\{\#3\}}{gregoriotex-spaces.tex}
+Macro to change one of Gregorio\TeX’s distances.  This function will check to make sure the distance you are trying to change exists first.
 
 \begin{argtable}
-  \#1 & string & element whose styling is to be changed (see list above for options)\\
-  \#2 & \TeX\ code & the code necessary to turn on the styling\\
-  \#3 & \TeX\ code & Optional.  The code necessary to turn off the styling (e.g. if the code to turn on the styling contains a \verb=\begin{environment}= then the code to turn it off must have the matching \verb=\end{environment}=.
+  \#1 & string & The name of the distance to be changed.  See \nameref{distances} below.\\
+  \#2 & string & The distance in string format.  \textbf{Note:} You cannot use a length register for this argument.  You \emph{must} use a string because of the way that Gregorio\TeX\ handles spaces.\\
+  \#3 & 0 & Distance will not scale when staff size is changed.\\
+  & 1 & Distance will scale when staff size is changed.
 \end{argtable}
 
-Examples:\par\medskip
+\macroname{\textbackslash grescaledim}{\{\#1\}\{\#2\}}{gregoriotex-spaces.tex}
+Macro to turn on or off scaling with the staff size for a particular distance.
+
+\begin{argtable}
+  \#1 & string & The name of the distance for which scaling it to changed.  See \nameref{distances} below.\\
+  \#2 & \texttt{yes}/\texttt{true}/\texttt{on} & Scale the distance when changing the size of the staff.\\
+  & \textit{other} & Do not scale the distance when changing the size of the staff.
+\end{argtable}
+
+\macroname{\textbackslash greloadspaceconf}{\{\#1\}}{gregoriotex-spaces.tex}
+Macro to load a space configuration file.  Space configuration file names have the format \verb=gsp-identifier.tex= and must be in the same directory as your project or in your texmf directory.  This function if \texttt{gre} prefixed to highlight that it only loads distances for Gregorio\TeX.
```

Comment:
"This function if \texttt{gre} prefixed" should be "This function is \texttt{gre}-prefixed"